mantidproject/mantid
# Fix memory leak in LoadSassena

Description of work.

Clang-Tidy found two [potential leaks of memory pointed to by 'buf'](http://builds.mantidproject.org/view/Static%20Analysis/job/clang_tidy/228/warnings13Result/new/) in LoadSessena. The easiest way to address this is to use std::vector<double>.

**To test:**



Reviewing changes should be sufficient. Optionally, run Clang-Tidy on LoadSessena.cpp with the  
`clang-analyzer-cplusplus.NewDeleteLeaks` check enabled.

This PR has no associated GitHub issue.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
